community.md

```diff
@@ -144,6 +144,18 @@ If you send us an email with a code snippet, make sure that:
 3. you keep formatting when pasting code in order to keep the code readable
 4. there are enough import statements to avoid ambiguities
 
+## Slack
+
+You can join the [Apache Flink community on Slack.](https://join.slack.com/t/apache-flink/shared_invite/zt-19uavkkkw-qwMimi9gawIv9Fz9xFuqrw)
```

Comment:
I wonder if it makes sense to use the ASF URL shortener: https://s.apache.org/
It allows updating the underlying link (however only for the owner of the id) .. but it would still be easier than merging a change to flink-web after 100 invites.

Reply:
Shall we add a note here for how to reach out to the community that the invitation link hit the limitation?